traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -270,7 +270,7 @@ FULL OUTER JOIN deliveryservice_server dss ON dss.server = s.id
 		if err = rows.StructScan(&s); err != nil {
 			return nil, []error{fmt.Errorf("getting servers: %v", err)}, tc.SystemError
 		}
-		if user.PrivLevel < auth.PrivLevelAdmin {
+		if user.PrivLevel < auth.PrivLevelOperations {
```

Comment:
I don't really agree with changing the API underneath people, when it's always required Admin. But I understand the difficulty of fixing the Portal to handle this.

But at the least, the documentation needs updated to match. The docs currently say
`The password of the of the lights-out-management user (displays as ** unless you are an ‘admin’ user).`

Reply:
ok, i'll update